File: daily-coding/coding-bat/rotate_left3.py
# Given an array of ints length 3, return an array with the elements "rotated left" so {1, 2, 3} yields {2, 3, 1}.

def rotate_left3(nums):
    my_list = nums[1::]
    my_list.append(nums[0])
    return my_list

File: daily-coding/coding-bat/java/rotateLeft3.java
// Given an array of ints length 3, return an array with the elements "rotated left" so {1, 2, 3} yields {2, 3, 1}.

public int[] rotateLeft3(int[] nums) {
  int[] myArr = nums;
  int last = nums[0];

  for(int i = 0; i < 2; i++){
    myArr[i] = nums[i+1];
  }
  myArr[2] = last;

  return myArr;
}
